src/v2/view-builder/views/signin/SignInWithDeviceOption.js

```diff
@@ -4,17 +4,37 @@ import hbs from 'handlebars-inline-precompile';
 export default View.extend({
   className: 'sign-in-with-device-option',
   template: hbs`
-    <div class="okta-verify-container"></div>
-    <div class="separation-line"><span>OR</span></div>
+    <div class="okta-verify-container">
```

Comment:
would it be like this?
```

<div class="okta-verify-container">
     {{#if signInWithDeviceIsRequired}}
       <div class="signin-with-ov-description">
         {{i18n code="oktaVerify.description" bundle="login"}}
       </div>
     {{else}}
     <div class="separation-line">
       <span>{{i18n code="authbutton.divider.text" bundle="login"}}</span>
     </div>
     {{/if}}
</div>
```

Reply:
that's not the current DOM structure. div `.okta-verify-container` and div `.separation-line` are in parallel

Reply:
oh..missed that.
when `signInWithDeviceIsRequired` is false, you got empty `okta-verify-container` container, why not just get rid of completely.

Reply:
No, when it is false, another button is added from line 22.

Reply:
I see. try https://handlebarsjs.com/guide/builtin-helpers.html#unless . probably can simplify a bit.